doc/sphinx/Makefile

```diff
@@ -41,6 +41,13 @@ copy-docs: clean-copy-docs
 	cp  ../release/quickReference.pdf     source/language/quickReference.pdf
 	cp  ../release/chapelLanguageSpec.pdf source/language/chapelLanguageSpec.pdf
 
+primers: clean-primers
+	@echo
+	@echo "Generating primers from "'$$CHPL_HOME'"/test/release/examples to "'$$CHPL_HOME'"/doc/sphinx/source/primers"
+	./chpl2rst.py ../../test/release/examples/primers/*.chpl --output=rst --prefix=source/primers
```

Comment:
Are you referring to the release tarball case, or just think there should be a test in general? This Makefile has other assumptions about relative paths as well - if we add this check for `$CHPL_HOME`, we might as well use it across the whole file.